development_workflow.md

```diff
@@ -636,25 +660,21 @@ If Abigail is an owner of the NEST repository, she can also add the label “doc
 
 If no one objects, the release manager merges the Pull Request into the public version of NEST.
 
-Alex wants to fix a bug in the NEST kernel
-------------------------------------------
+## Alex wants to fix a bug in the NEST kernel
```

Comment:
The workflows from here on down are unfinished work. I think we should remove them.